### PR TITLE
feature/add .npmrc file in dapp

### PR DESCRIPTION
**Pull request by Le-Caignec**

### CODE REVIEW

File: dapp/.npmrc [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# .npmrc
+engine-strict=true
```
Comment (cedric25) [inline dapp/.npmrc:2]:
```suggestion
engine-strict=true
```

**Comment by cedric25** [comment]
It looks like it does what we need: stopping `npm install` in case of an unsupported node version:

Capture d’écran 2023-11-06 à 17 00 47

Also I would suggest to add an `.nvmrc` file that indicates the required node version:
```
14
```
Quite handy for people using something like nvm, n or fnm.

File: dapp/.nvmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+>=14.0.0 <15.0.0
```
Comment (cedric25) [inline dapp/.nvmrc:1]:
```suggestion
14
```

**Review by cedric25 — APPROVED** [review]

File: dapp/.nvmrc (hunk 2 above)
Comment (Le-Caignec) [reply dapp/.nvmrc]: done

**Pull request merged**